documentation/asciidoc/services/connect/install.adoc

```diff
@@ -1,7 +1,7 @@
 [[install-connect]]
 == Install
 
-To begin installation, open a Terminal window. Run the following command to update your system and packages:
+If Connect isn't already installed in your version of Raspberry Pi OS, open a Terminal window. Run the following command to update your system and packages:
```

Comment:
@spl237 suggested:

> Under Install, it might be worth mentioning that it can be installed from the Recommended Software GUI app, for those who don’t like terminal things. Only installs the full version, not lite.

Reply:
Added!